src/plugins/embeddable/public/lib/test_samples/embeddables/contact_card/contact_card_embeddable.tsx

```diff
@@ -29,6 +29,7 @@ export interface ContactCardEmbeddableOutput extends EmbeddableOutput {
 
 export interface ContactCardEmbeddableOptions {
   execAction: UiActionsStart['executeTriggerActions'];
+  outputOverrides?: Partial<ContactCardEmbeddableOutput>;
```

Comment:
Having a default description breaks many of the other tests because it forces the panel title to show. So it made more sense to create a new embeddable factory that is used in tests that need a description.